apps/app/pages/admin/lectures/index.tsx

```diff
@@ -113,10 +116,28 @@ function Lectures() {
       })
       .catch(() => {});
   };
+
+  console.log(selectedLectures);
+  console.log(selectedLectures[0]?.notes == null);
```

Comment:
These are still here 😅

Reply:
I forgot to remove those :sweat_smile:  , i think it's everything as you wanted now